src/views/Content/Recipe/Recipe.js

```diff
@@ -8,13 +8,14 @@ import './Recipe.css';
  */
 
 function Recipe(props) {
+    const { item, entities, id } = props;
     let perSecond = 0;
     return (
-        <div className="recipe" id="viewedItem">
-            Engine unit (amount per second input)
-            <div>Gear (amount) {perSecond}</div>
-            <div>Pipe (amount) (amount per second)</div>
-            <div>Steel plate (amount) (amount per second)</div>
+        <div className="recipe" id={id} >
+            {entities.get(item) ? entities.get(item).name : "startup"} (id === "viewedItem" ? input : amount per second)
```

Comment:
I would not do the lookup twice. Either assign as a variable, or use .has() instead of get(). Might do "loading.." instead of "startup"